html_table_parser/parser.py

```diff
@@ -52,7 +56,7 @@ def handle_endtag(self, tag):
             self._in_th = False
 
         if tag in ['td', 'th']:
-            final_cell = " ".join(self._current_cell).strip()
+            final_cell = "".join(self._current_cell).strip()
```

Comment:
You're referring to the question above, aren't you? I would not put this functionality inside the parser since it's just not parsing of an html table.

Couldn't you go over the parsed table later and unescape everything then?